utils/users.js

```diff
@@ -81,9 +81,19 @@ const getParticipantUserIds = async (participantArray) => {
   }
 };
 
+/**
+ * Returns a random object from the array of colors to user
+ * @param cardColorArray {array} : color containing array
+ * @returns userColorIndex number : index between the range 0 to cardColorArray.length
+ */
+const addColorsProperty = (cardColorArray) => {
+  return Math.floor(Math.random() * (cardColorArray.length - 0) + 0);
+};
```

Comment:
Can we make it more generic? 

something like 
`const getRandomItem = (array = []) => { return ...}

Reply:
why should we make it generic, when we have a specific use case for the function?

Reply:
Good point out. We are making it generic as to be able to use it in other places in future .